MoltenVK/MoltenVK/GPUObjects/MVKDescriptor.mm

```diff
@@ -148,7 +148,11 @@
                 MVKBuffer* buffer = (MVKBuffer*)bufferInfo.buffer;
                 bb.mtlBuffer = buffer->getMTLBuffer();
                 bb.offset = buffer->getMTLBufferOffset() + bufferInfo.offset;
-				bb.size = (uint32_t)buffer->getByteCount();
```

Comment:
@js6i Is there a use-case where this is actually causing problems?

When it hits Metal, `MVKMTLBufferBinding.size` is only used when setting inline bytes, not when using an actual `VkBuffer/MTLBuffer`, so `VK_WHOLE_SIZE` should have no applicability.

@cdavis5e `MVKMTLBufferBinding.size` was introduced in support of SPIRV `OpArrayLength`. Your thoughts on this?

Reply:
Not that I know of. I was confused by this line, but didn't see the value used anywhere. If it's intended to stay for some reason, I'd put a comment there.

Reply:
That is used to set the buffer size passed down to the shader. See MoltenVK/MoltenVK/Commands/MVKCommandEncoderState.mm:585 (or thereabouts). This change appears to be correct.